library/swift/RetryPolicy.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+/// Rules that may be used with `RetryPolicy`.
+/// See the `x-envoy-retry-on` Envoy header for documentation.
+@objc
+public enum RetryRule: Int {
```

Comment:
Same thoughts as on the above enum implementation.

Reply:
https://github.com/lyft/envoy-mobile/pull/242#discussion_r302692922